src/TestTask/components/Stats/Stats.stories.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import { withKnobs } from '@storybook/addon-knobs'
+import { storiesOf } from '@storybook/react'
+import { useAddonState } from '@storybook/store'
+
+import Stats from './Stats'
+
+storiesOf('test/components/Stats', module)
+  .addDecorator(withKnobs)
+  .add('default', () => {
+    const [data, setData] = useAddonState<Array<{ timestamp: number; value: number }>>('time', [])
+
+    React.useEffect(() => {
```

Comment:
I think you should use storybook hooks inside stories

Reply:
also, you're setting data only once, hooks usage seems to be unnecessary

Reply:
Noted.